src/convertToObject.js

```diff
@@ -5,8 +5,20 @@
  *
  * @return {object}
  */
+
 function convertToObject(sourceString) {
-  // write your code here
+  return sourceString
```

Comment:
There is a syntax error here. The division operator `/` is mistakenly used instead of a dot `.` to chain the method calls. This will cause a runtime error. Replace `/` with `.` to correctly chain the `split` method to `sourceString`.